aws/aws_eip.go

```diff
@@ -2,36 +2,45 @@ package aws
 
 import (
 	"github.com/jmcgill/formation/core"
-	//"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws"
+	// "github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/davecgh/go-spew/spew"
 )
 
 type AwsEipImporter struct {
 }
 
 // Lists all resources of this type
```

Comment:
Can you add a comment about the known issue with this + a link to the issue you created in Github?